src/app/components/Notifications/__snapshots__/NotificationsDropdown.test.tsx.snap

```diff
@@ -843,7 +843,7 @@ exports[`Notifications > should open and close transaction details modal 1`] = `
                                   class="flex h-full items-center space-x-1"
                                 >
                                   <span
-                                    class="whitespace-nowrap text-sm"
+                                    class="whitespace-nowrap text-sm text-[1rem]"
```

Comment:
it's a snapshot, but I have two issues with it:
- `text-[1rem]` being used instead of `text-base`
- this illustrated that there is an overlap in text classes, e.g. `text-sm text-[1rem]` works now because the 1rem overwrites it, but that is not a given. This will be better to use twmerge with to avoid the duplicate classes